2022.06.1/alpine3.16/Dockerfile

```diff
@@ -25,7 +25,9 @@ RUN buildDeps=' \
     \
     && wget ${url}.asc -O ${tmpdir}/rakudo.tar.gz.asc \
     && wget $url -O ${tmpdir}/rakudo.tar.gz \
-    && gpg --batch --keyserver $keyserver --recv-keys $keyfp \
+    && wget $pubkeyurl -O ${tmpdir}/${keyfp}.asc \
+    && gpg --import ${tmpdir}/${keyfp}.asc \
+    && gpg --check-signatures $keyfp \
```

Comment:
Doesn't this just verify that `$keyfp` is in the set of keys imported?
(I think this still leaves it open to an extra key getting injected into the file on the server which will then satisfy `--batch --verify` as long as the original key exists in the imported file too)

In cases like this previously I've suggested something like `gpg --import ... && gpg --export FULL-FINGERPRINT > some-new-file && rm -rf GNUPGHOME && gpg --import some-new-file` :see_no_evil: